resource/plugins/unreal/python/publisher/post_finalizers/unreal_dependencytrack_publisher_post_finalizer.py

```diff
@@ -0,0 +1,137 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+import json
+
+import ftrack_api
+
+from ftrack_connect_pipeline import constants as core_constants
+from ftrack_connect_pipeline_unreal import plugin
+from ftrack_connect_pipeline_qt import constants as qt_constants
+
+from ftrack_connect_pipeline_unreal.utils import (
+    custom_commands as unreal_utils,
+)
+
+
+class UnrealDependencyTrackPublisherFinalizerPlugin(
+    plugin.UnrealPublisherPostFinalizerPlugin
+):
+    '''Plugin for finalizing the Unreal publish process for a dependency'''
```

Comment:
Add more info here, what is the difference between this one and the unreal_dependencies_publisher_post_finalizer.py????